### PR TITLE
Created a landing page

### PR DESCRIPTION
**Pull request by olofvi**
# PT link: https://www.pivotaltracker.com/n/projects/2113540
--------

## Created a landing page

What we have learned

- We learned how to create a database and routes
- Practised TDD with cucumber
- We remembered to NOT work a the develop branch

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project version="4">
+  <component name="VcsDirectoryMappings">
+    <mapping directory="$PROJECT_DIR$" vcs="Git" />
+  </component>
+</project>
```
Comment (tochman) [inline .idea/vcs.xml]: Hmm... More `.idea` files...

File: config/routes.rb [hunk 2]
```diff
@@ -1,3 +1,5 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
+  get 'landing/index'
+
+  root controller: :landing, action: :index
```
Comment (tochman) [inline config/routes.rb]: These two routes are redundant

**Comment by olofvi** [comment]
Fixed the .idea and the routes :)

**Review by tochman — APPROVED** [review]
Great start.

**Pull request merged**